src/meshapi/admin/utils.py

```diff
@@ -18,7 +18,10 @@ def get_admin_url(model: Model, site_base_url: str) -> str:
         reverse(
             f"admin:{content_type.app_label}_{content_type.model}_change",
             args=(model.pk,),
-        ),
+        ).replace(
+            "panel/", ""
+        ),  # Strip "panel" out of admin URL. "Panel" views are meant to be shown
+        # inside the admin panel iframe, and likely the user does not want that
```

Comment:
Is there a better way to do this? `reverse()` is the canonical way to get a URL to something in Django, so if it's not returning the correct value that's not great

Reply:
I don't like it either. That was my "get the fucking thing working" answer. I still need to root-cause it.

Reply:
I think it's because that's how I've configured `admin.site.urls`. I might need to play with that some more.

Reply:
One option: I can put the admin panel at a new URL (like "admin/panel" maybe) and do the redirect kinda the other way around. It'll make sense on my next push.